### DISCUSSION TITLE
Will Discussions will be available for private repositories ?

### DISCUSSION THREAD

**Discussion by scorsi** [community/community#40] [Discussions] [answered]
Hello,

We're very interested into that feature. We would like to have this feature enabled in our private repository to talk about features to add/how to do that stuff/where is that stuff/... Instead of having a Slack Channel which is now too heavy.

Thanks !

**Comment by becca** [comment, marked as answer]
Hello there! Right now Discussions is only available for public repositories.

**Reply by gotom22** [reply]
Is this referring to repos specifically? I see Discussions in our private organisation.

**Reply by becca** [reply]
Are you referring to Team Discussions potentially, @gotom22? https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-team-discussions

**Reply by gotom22** [reply]
yes, sorry - when I said discussions at org level I meant team discussions. So basically: will team discussions get the same features as repo discussions, eventually?

**Reply by gotom22** [reply]
yes, sorry - when I said discussions at org level I meant team discussions. So basically: will team discussions get the same features as repo discussions, eventually?
